test/main.ts

```diff
@@ -733,3 +733,24 @@ test('parseJson option with promise.json() shortcut', async t => {
 
 	await server.close();
 });
+
+test('stringifyJson option with request.json()', async t => {
+	const server = await createHttpTestServer({bodyParser: false});
+	server.post('/', async (request, response) => {
+		const body = await parseRawBody(request);
+		t.is(body, '{"data":{"hello":"world"},"extra":"extraValue"}');
```

Comment:
Would be more readable as an object passed to `JSON.stringify`.

Reply:
Just move the `JSON.stringify({
			data,
			extra: 'extraValue',
		})` part to the top and use it in this check too.